manifests/b/Bluesound/BluOSController/3.18.1.1675/Bluesound.BluOSController.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Bluesound.BluOSController
+PackageVersion: 3.18.1.1675

```

```suggestion
PackageVersion: 3.18.1
```
Must match control panel